includes/class-woopay-tracker.php

```diff
@@ -151,10 +152,17 @@ public function maybe_record_wcpay_shopper_event( $event, $data = [] ) {
 			$event = self::$user_prefix . '_' . $event;
 		}
 
-		$is_admin_event      = false;
-		$track_on_all_stores = true;
```

Comment:
Because `track_on_all_stores` is not passed, page views are not recorded when WooPay is disabled.

Test instructions:
1. Disable WooPay.
2. Add a breakpoint [here](https://github.com/Automattic/woocommerce-payments/blob/67a5a49a21b6afa1feef628122b2cad494c8573a/includes/class-woopay-tracker.php#L290)
3. Go to a product page.
4. Breakpoint is not reached.

Reply:
But the page views are being recorded on the front-end now right? I can see requests from the `frontend-tracks` file with the correct `recordUserEvent` on the DevTools Network tab when WooPay is disabled.

Reply:
> But the page views are being recorded on the front-end now right?

Correct but, all what `recordUserEvent` does is create another ajax request. When processing the request it will eventually reach `should_enable_tracking`.

Previously, because `$track_on_all_stores` was `true` for events recorded through `maybe_record_wcpay_shopper_event`, `should_enable_tracking` correctly returned `true` for these events regardless whether WooPay was enabled or not. But now, since we no longer have `track_on_all_stores`, these events only get recorded when WooPay is enabled.

Reply:
@malithsen nice catch, fixed on https://github.com/Automattic/woocommerce-payments/pull/9249/commits/c20aef6f86701a2a169bcb886f7b7f90ce4fb933.